app/src/main/java/org/fossasia/phimpme/gallery/data/HandlingAlbums.java

```diff
@@ -312,6 +317,27 @@ public void sortAlbums(final Context context) {
     }
   }
 
+  public void selectAllPhotosUpToAlbums(int targetIndex, AlbumsAdapter adapter) {
+    int indexRightBeforeOrAfter = -1;
+    int indexNow;
+    for (Album selectedAlbum : selectedAlbums) {
+      indexNow = dispAlbums.indexOf(selectedAlbum);
+      if (indexRightBeforeOrAfter == -1) indexRightBeforeOrAfter = indexNow;
+
+      if (indexNow > targetIndex) break;
```

Comment:
Properly intend this line to another line pls.

Reply:
Done the changes